tutorials/requirements.txt

```diff
@@ -10,5 +10,9 @@ pandas>=1.5.2
 dask[distributed]
 psutil
 s3fs
+firefly_client
+jupyter_firefly_extensions
```

Comment:
Couldn't help but notice: the pip packages names for both of these use `-` instead of `_` - I ran into similar confusion earlier and couldn't install the package, I'm surprised to see it's working in CI with `_`

Reply:
hahh, interesting. I suppose pip replaces them at install, but indeed I should declare them as they are named on pypi.